### DISCUSSION TITLE
test_bimodal using bed file - empty results

### DISCUSSION THREAD

**Discussion by emjblake** [nloyfer/wgbs_tools#63] [Q&A] [answered]
I am unable to generate test_bimodal results when using `--bed_file`; it does, however, work with `--region`. Here is the content of my test.bed.gz file: 

```
chr1    9869    11868   1       130
chr1    10010   12009   1       134
chr1    17437   19436   244     298
```

Command with `-L` option:

`$wgbstools test_bimodal -L test.bed.gz sample.pat.gz`

_No results returned_

Command with `-r` option for comparison:

`$wgbstools test_bimodal -r chr1:9869-11868 sample.pat.gz`

Results:
```
LL: -7.578929530800584 | 3 reads | 66 observed | BPI: 0.9236031051597491
LL: -10.5789295308006 | 3 reads | 66 observed | BPI: 0.8950005801519093
pvalue: 1.000e+00
```

Any ideas?

**Comment by yonniejon** [comment, marked as answer]
Hi!
Sorry about that. That is a problem with the documentation as it is not clear. test_bimodal using a bed file as input only prints out those regions which show statistically significant bimodal methylation (after FDR correction). It prints the bed file region and the corrected p-value. This should be documented and I will make a fix for this. Besides that I can add an option to print the p-value of all regions, and not just those which are bimodal.

**Reply by emjblake** [reply]
Thanks for clarifying! Is the corrected p-value threshold 0.05? It would be great to get the p-value for all regions in the bed file, not just significant regions - thanks!
